client/api/events.ts

```diff
@@ -158,6 +159,9 @@ function create(updates: Partial<IEventItem>): Promise<Array<IEventItem>> {
         .then((response) => {
             const events = modifySaveResponseForClient(response);
 
+            // Update redux store, so changes are reflected in associated_event field
+            planningApi.redux.store.dispatch<any>(eventsApi.receiveEvents([response]));
```

Comment:
shouldn't you call modifySaveResponseForClient before pushing it to redux store?